conf/report/report.css

```diff
@@ -3,17 +3,36 @@ body {
 }
 
 .headline {
-    text-align: center;
+  width: 100%;
+  text-align: center;
+  position: fixed;
+  display: table-row;
 }
 
 .headline a {  /* Only subtly hint that this is a link */
-    color: #00006f;
-    text-decoration: none;
+  color: #00006f;
+  text-decoration: none;
+}
+
+label {
+  position: fixed;
+  right: 0;
```

Comment:
This causes the search input to be overlapped with the table, we don't want the position to be fixed, we want to be able to scroll down and leave it behind at this point

Reply:
I understand, however, when the headline is fixed and the search box is moving, they were overlapping on the x-axis. Maybe it was because I didn't use the right approach to fix the headline. I'll look into this, thanks!

Reply:
Oh, okay, I understand what you wanted to do here then - unfortunately it causes the issues we can see when scrolling down.

It's nice that you tried to fix it, but please keep one feature per PR - otherwise it is hard to tell which code change is expected to achieve what.

And as for this change, maybe we could move the horizontal scroll to be inside the table (as opposed to scrolling the whole page like it happens now): https://datatables.net/examples/basic_init/scroll_x.html

(but let's do this separately, not as a part of this PR)

Reply:
Sure! I'll work on only fixing the variable width columns for now then. I've also just realised that the column width still changes on sorting, so I'll look into it and push the changes 😄